Tests/Marketplace/copy_and_upload_packs.py

```diff
@@ -347,14 +347,21 @@ def main():
 
     # Get the successful and failed packs file from Prepare Content step in Create Instances job if there are
     packs_results_file_path = os.path.join(os.path.dirname(packs_artifacts_path), BucketUploadFlow.PACKS_RESULTS_FILE)
-    pc_successful_packs_dict, pc_failed_packs_dict, pc_successful_private_packs_dict, \
+    pc_successful_packs_dict, pc_successful_uploaded_dependencies_zip_packs_dict, \
+        pc_failed_packs_dict, pc_successful_private_packs_dict, \
         pc_uploaded_images = get_upload_data(packs_results_file_path, BucketUploadFlow.PREPARE_CONTENT_FOR_TESTING)
 
     logging.debug(f"Successful packs from Prepare Content: {pc_successful_packs_dict}")
+    logging.debug(f"Successful uploaded dependencies zip packs from Prepare Content: "
+                  f"{pc_successful_uploaded_dependencies_zip_packs_dict}")
     logging.debug(f"Failed packs from Prepare Content: {pc_failed_packs_dict}")
     logging.debug(f"Successful private packs from Prepare Content: {pc_successful_private_packs_dict}")
     logging.debug(f"Images from Prepare Content: {pc_uploaded_images}")
 
+    # we want to copy also packs that successfuly uploaded their dependencies zip, thus unifying
+    # pc_successful_packs_dict and pc_successful_uploaded_dependencies_zip_packs_dict
+    pc_successful_packs_dict = pc_successful_packs_dict | pc_successful_uploaded_dependencies_zip_packs_dict
```

Comment:
I'm not sure that unifing them is good as we are passing later the `pc_successful_packs_dict` to the `verify_copy` and it will include also the success dependencies zip, maybe the solution is when checking the success list check the success dependencies zip list too.

Also, the `verify_copy` don't fail because that also packs that were just copied the dependencies zip were marked with SUCCESS and not with SUCCESS_CREATING_DEPENDENCIES_ZIP_UPLOADING, this will also cause printing the packs that were just uploaded with dependencies to be under successful packs, I think we need to mark them correctly here as well.